Core/Services/StitchCoreRemoteMongoDBService/Sources/StitchCoreRemoteMongoDBService/Sync/Internal/DataSynchronizer.swift

```diff
@@ -420,7 +419,7 @@ public class DataSynchronizer: NetworkStateDelegate, FatalErrorListener {
     }
 
     private func syncRemoteChangeEventToLocal(nsConfig: NamespaceSynchronization,
-                                              docConfig: CoreDocumentSynchronization,
+                                              docConfig: inout CoreDocumentSynchronization,
```

Comment:
[nit] `CoreDocumentSynchronizationConfig` is a `class`, so you can get rid of the `inout` qualifier on this method, remove the `&` in the invocations of this method, and make the `docConfig` local var a `let` in the places you're calling this method